yb-voyager/src/srcdb/source.go

```diff
@@ -86,12 +87,28 @@ func (s *Source) IsOracleCDBSetup() bool {
 
 func (s *Source) ApplyExportSchemaObjectListFilter() {
 	allowedObjects := utils.GetSchemaObjectList(s.DBType)
-	if s.StrExportObjectTypesList == "" {
+
+	if s.StrExportObjectTypesList == "" && s.StrExcludeObjectTypesList == "" {
 		s.ExportObjectTypesList = allowedObjects
 		return
 	}
-	expectedObjectsSlice := strings.Split(s.StrExportObjectTypesList, ",")
+
+	if s.StrExcludeObjectTypesList != "" {
+		excludedObjectsSlice := utils.CsvStringToSlice(s.StrExcludeObjectTypesList)
+		s.ExportObjectTypesList = lo.Filter(allowedObjects, func(objType string, _ int) bool { return !utils.ContainsString(excludedObjectsSlice, objType) })
+		filteredObjects := lo.Filter(excludedObjectsSlice, func(objType string, _ int) bool { return !utils.ContainsString(allowedObjects, objType) })
+		if len(filteredObjects) > 0 {
+			utils.PrintAndLog("Ignoring invalid object types: %s\n", strings.Join(filteredObjects, ", "))
```

Comment:
Maybe just exit and ask the user to provide a proper list and display the allowed list.